README.md

```diff
@@ -40,7 +40,21 @@ For local development, we recommend to use the following setup which will enable
 pip install --upgrade pip
 git clone git@github.com:awslabs/syne-tune.git
 cd syne-tune
-pip install -e .[extra]
+pip install -e '.[extra]'
+```
+
+You may want to use environment management system such as Conda:
+```bash
+conda create -n syne-tune python=3.8
+conda activate syne-tune
+# run install comands here
+```
+
+To run unit tests, simply run `pytest` in the root of this repository.
+
+To run all tests whose name begins with `test_config_space_for_sampling`, you can use the following
+```bash
+pytest -k test_use_active_space_for_sampling
```

Comment:
It would be nice to use a more representative example, for instance `test_schedulers_api.py` that people will often need.

Reply:
Used example from test_schedulers_api.py .